src/core/sys/posix/sys/ioctl.d

```diff
@@ -323,13 +323,7 @@ else version (OSX)
     import core.sys.posix.termios; // termios
     import core.sys.posix.sys.time; // timeval
 
-    struct winsize
```

Comment:
oh, I see what you mean, I didn't realize the context of the file in the diff. cool.